library/swift_disk.py

```diff
@@ -59,21 +84,21 @@ def main():
     # write fstab
     try:
         with open('/etc/fstab', 'a') as f:
-            f.write("UUID=%s /srv/node/%s1 xfs noatime,nodiratime,nobarrier,logbufs=8 0 0\n" % (fsuuid, dev))
+            f.write("UUID=%s %s xfs noatime,nodiratime,nobarrier,logbufs=8 0 0\n" % (fsuuid, mount_point))
     except Exception, e:
         module.fail_json(msg="failed to update fstab: %s" % e)
 
     # mount point
-    os.makedirs('/srv/node/%s1' % dev)
+    os.makedirs(mount_point)
```

Comment:
At some point we should probably remove the fstab munging bit of these libraries and use Ansible's module for this. The module here should just prep the disks to be mounted, that's easier to test for work already complete.